cheatsheets/Vim.rb

```diff
@@ -277,8 +277,8 @@
 
             * `[c]` Confirm each substitution. Vim positions the cursor on the matching string. You can type:
               * `y`      to substitute this match
+              * `l`      to substitute this match and then quit ("last")
               * `n`      to skip this match
-              * `<Esc>`   to skip this match
```

Comment:
This was wrong, `<Esc>` is the same as `q`. But it seems redundant to have both, especially since `<Esc>` isn't listed as an option when making a substitution:

```
replace with foo (y/n/a/q/l/^E/^Y)?
```